cypress/support/config/settings.js

```diff
@@ -4316,11 +4316,11 @@ module.exports = () => ({
         environments: {
           live: {
             paths: ['/serbian/lat/popular/read'],
-            enabled: false,
+            enabled: true,
           },
           test: {
             paths: ['/serbian/lat/popular/read'],
-            enabled: false,
+            enabled: true,
           },
           local: {
             paths: ['/serbian/lat/popular/read'],
```

Comment:
For completeness, this test on local should be 'true', too.

Reply:
🤦 I missed that